src/Razor/test/Microsoft.VisualStudio.LanguageServer.ContainedLanguage.Test/VirtualDocumentFactoryBaseTest.cs

```diff
@@ -52,6 +52,7 @@ public void TryCreateFor_IncompatibleHostDocumentBuffer_ReturnsFalse()
             // Assert
             Assert.False(result);
             Assert.Null(virtualDocument);
+            virtualDocument.Dispose();
```

Comment:
Hmm, do we need a `method(..., using out var virtualDocument)` syntax in C# or does something like that already exist❔